clients/src/main/java/org/apache/kafka/common/protocol/SecurityProtocol.java

```diff
@@ -26,6 +26,8 @@
     PLAINTEXT(0, "PLAINTEXT"),
     /** SSL channel */
     SSL(1, "SSL"),
+    /** SASL authenticated, non-encrypted channel */
+    PLAINTEXTSASL(2, "PLAINTEXTSASL"),
```

Comment:
@ijuma enabling qop on sasl proven to cause lot of perf issues . It was discussed before hence the reason I went with proposal ssl+sasl.